src/main/java/ui/components/pickers/AssigneePickerState.java

```diff
@@ -0,0 +1,84 @@
+package ui.components.pickers;
+
+import util.Utility;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class AssigneePickerState {
+    private List<PickerAssignee> currentAssigneesList;
+
+    public AssigneePickerState(List<PickerAssignee> assignees) {
+        currentAssigneesList = cloneList(assignees);
+    }
+
+    public AssigneePickerState(List<PickerAssignee> assignees, String userInput) {
+        this(assignees);
+        processInput(userInput);
+    }
+
+    private List<PickerAssignee> cloneList(List<PickerAssignee> sourceList) {
```

Comment:
Method name could be a bit more specific, since it only applies to lists of PickerAssignees. `cloneAssignees`? Also can be static.